### DISCUSSION TITLE
Question on upscaling the stack in hubs-cloud and pre-made rooms

### DISCUSSION THREAD

**Discussion by elmau** [mozilla/hubs#3079] [Hubs Cloud Questions] [answered]
Has anyone done this? making several rooms beforehand an event, update the stack as is stated on the documents to a larger server and then the rooms remain intact with their codes? or we have to generate the rooms after the stack updates?

We need to make several rooms a couple of days before the event, and we want to separate users giving them different room codes.

**Comment by robinkwilson** [comment, marked as answer]
Good question, after a stack update, the rooms stay persistent. They are saved in your database across updates. Don't worry!

**Reply by robinkwilson** [reply]
Though if they don't... please let us know, because that would be a bug!

**Reply by elmau** [reply]
What about downscaling during an event?
